src/components/questions_answers/QnA_Components/MoreQuestions.jsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import Question from './Question';
+
+export default function MoreQuestions({ questions }) {
+  const [collapseQuestions, setCollapseQuestions] = useState(true);
+  return (
+    <div>
+      {collapseQuestions ? (
+        <div>
+          <button
+            type="button"
+            onClick={() => {
+              setCollapseQuestions(!collapseQuestions);
+            }}
+          >
+            More Answered Questions
+          </button>
+        </div>
+      ) : (
+        <div>
+          {questions.slice(4).map((question) => (
+            <Question question={question} key={question.question_id} />
+          ))}
+          <div>
```

Comment:
Same question as above on MoreAnswers

Reply:
hi @arangotang , thank you for reviewing, I have made some changes and added some basic css, hope it look better now. Thanks!